packages/cnpj-gen/tests/Specs/CnpjGenerator.isolated.spec.php

```
@@ -0,0 +1,102 @@
+         * >
+         */
+        public array $callsArguments = [];
+
+        public function generate(
+            ?CnpjGeneratorOptions $options = null,
+            $format = null,
+            $prefix = null,
+            $type = null,
+        ): string {
+            $this->callsCount++;
+            $this->callsArguments[] = func_get_args();
```

PHPStan (2.1.54)

[warning] Property CnpjGeneratorCallsSpy::$callsArguments (list<array{Lacus\BrUtils\Cnpj\CnpjGeneratorOptions|null, mixed, mixed, mixed}>) does not accept non-empty-list<list<mixed>>.


(assign.propertyType)